allennlp/models/model.py

```diff
@@ -87,8 +101,70 @@ def get_metrics(self, reset: bool = False) -> Dict[str, float]:
         # pylint: disable=unused-argument,no-self-use
         return {}
 
+    def archive(self,                                       # pylint: disable=no-self-use
```

Comment:
yeah, I considered putting them in `models/archive.py`, my only rationale against it is that archiving functions might need access to internal model state, but for now (when we're just archiving based on what's on disk) I guess we don't. I'll move them out